demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -1043,9 +1042,7 @@ def _find_pack_dependencies(pack_id: str, id_set: dict, verbose: bool,
         return pack_dependencies
 
     @staticmethod
-    def build_all_dependencies_graph(pack_ids: list,
-                                     id_set: dict,
-                                     verbose: bool,
+    def build_all_dependencies_graph(pack_ids: list, id_set: dict, verbose: bool = False,
```

Comment:
```suggestion
    def build_all_dependencies_graph(
        pack_ids: list,
        id_set: dict, 
        verbose: bool = False,
```
Helping with vertical readability. Like what you did in the 'verbose' section